tst/testinstall/declarefunction.tst

```diff
@@ -0,0 +1,45 @@
+#@local name
+
+gap> START_TEST("declarefunction.tst");
+gap> DeclareGlobalFunction("testfunctionA");
+gap> testfunctionA;
+function( args... ) ... end
+gap> Print(testfunctionA,"\n");
+function ( args... )
+    <<kernel code>> from the global function "testfunctionA" is not yet define\
+d:
+end
+gap> InstallGlobalFunction(testfunctionA, x -> x);
+gap> testfunctionA;
+function( x ) ... end
+gap> Print(testfunctionA,"\n");
+function ( x )
+    return x;
+end
+gap> name := List([1..100], x -> 'a');;
+gap> DeclareGlobalFunction(name);
```

Comment:
Minor nitpick (it is fine to ignore this!):

The one issue with this test is that re-running it in the same GAP session won't work, as then these global functions are already defined. Granted, that's not a serious issue most of the time, although I did spend some effort in the past trying to avoid such things.

If one wanted to, one could avoid it here by directly calling `NEW_GLOBAL_FUNCTION( name )`.

Reply:
fixed